src/managers/KeybindManager.cpp

```diff
@@ -1623,6 +1616,31 @@ SDispatchResult CKeybindManager::swapActive(std::string args) {
     if (PLASTWINDOW->isFullscreen())
         return {.success = false, .error = "Can't swap fullscreen window"};
 
+    if (args.starts_with("address:")) {
```

Comment:
I'd say do it the other way round. If it's not a direction, try to find the window with `g_pCompositor->getWindowByRegex`. That will do address, class, etc.

Just check if you didnt get nullptr or the active window